### PR TITLE
Czech eID card support

### PR DESCRIPTION
**Pull request by mkmayer**
Signed-off-by: Martin Mayer <martin.kelly.mayer@gmail.com>

### CODE REVIEW

**Comment by Muzosh** [comment]
Hello @mkmayer, how would you handle `unverifiedCertificate` verification in the authentication token on the back-end? Is this pull request intended only for limited number of entities in Czechia, that have access to intermediate certificate CAs? Or are there plans to release these state-issued public key certificates to the public?

**Comment by Muzosh** [comment]
I'm asking mainly because I would love to use Czech eIDs in non-state, Web-eID-enabled web applications myself :)

**Review by metsma — APPROVED** [review]

**Comment by metsma** [comment]
Please add signed off field to comment

**Comment by mkmayer** [comment]
> I'm asking mainly because I would love to use state-issued electronic identities from Czech eIDs in non-official, Web-eID-enabled web applications myself :)

Unfortunately I don't know this. I am currently using the web-eid-js component to sign the hash. At the same time, I send the certificate returned from the getCertificate method to our backend X509, and a colleague from the backend can handle it. I don't know, maybe it has imported certificates on the backend, I can't tell. I'm sorry I can't help.

**Comment by Muzosh** [comment]
Ok, thanks for the reply.

File: src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp [hunk 1]
```diff
@@ -99,6 +99,17 @@ inline fs::path belgianPkcs11ModulePath()
...
 #endif
 }
+	
+inline fs::path czechPkcs11ModulePath()
+{
+#ifdef _WIN32
+    return system32Path() / L"eopproxyp11.dll";
+#elif defined __APPLE__
+    return "/usr/local/lib/eOPCZE/libeopproxyp11.dylib"; 
+#else // Linux
```
Comment (klemensn) [inline src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp:109]: Would you mind adding preemptive OpenBSD support such that future ports for these card drivers won't require further changes to work?
```
#elif defined __OpenBSD__
    return "/usr/local/lib/libeopproxyp11.so";
```

That would go in line with #59.

**Review by mrts — APPROVED** [review]

**Comment by mrts** [comment]
Merged, this will be part of the upcoming 2.5.0 release - thanks a lot for your contribution once more!

**Pull request merged**